test/ruby/test_gc_compact.rb

```diff
@@ -452,4 +452,21 @@ def set_a
       assert_raise(FrozenError) { a.set_a }
     end;
   end
+
+  def test_moving_too_compex_generic_ivar
```

Comment:
Hi, little typo here (`compex` -> `complex`)

Reply:
Thanks for catching this. Fixed in 80a71bfb1cd58b4129e2b008fd0fccc7fc90a11e